tools/ktx/ktx_main.cpp

```diff
@@ -185,6 +192,24 @@ int _tmain(int argc, _TCHAR* argv[]) {
     //      // pbxproj file, so it can't be disabled in a generated project.
     //      // Remove these from the arguments under consideration.
 
+#if defined(_WIN32)
+    // Windows does not support UTF-8 argv so we have to manually acquire it
+    LPWSTR commandLine = GetCommandLineW();
+    LPWSTR* wideArgv = CommandLineToArgvW(commandLine, &argc);
+    std::vector<std::unique_ptr<_TCHAR[]>> utf8Argv(argc);
+    for (int i = 0; i < argc; ++i) {
+        int byteSize = WideCharToMultiByte(CP_UTF8, 0, wideArgv[i], -1, nullptr, 0, nullptr, nullptr);
+        utf8Argv[i] = std::make_unique<_TCHAR[]>(byteSize);
+        WideCharToMultiByte(CP_UTF8, 0, wideArgv[i], -1, utf8Argv[i].get(), byteSize, nullptr, nullptr);
+        argv[i] = utf8Argv[i].get();
+    }
+
+    // Set UTF-8 codepage for the console
+    if (!SetConsoleOutputCP(CP_UTF8)) {
```

Comment:
Will this work without error if the console is, e.g. Git for Windows Bash.

Reply:
Just tested it, it works as expected.